typegate/tests/utils/test.ts

```diff
@@ -193,6 +199,96 @@ export class MetaTest {
     return engine;
   }
 
+  async engineFromTgDeploy(path: string, cwd: string, tg?: any) {
+    const extension = extname(path);
+    let sdkLang: SDKLangugage;
+    switch (extension) {
+      case ".py":
+        sdkLang = SDKLangugage.Python;
+        break;
+      case ".ts":
+        sdkLang = SDKLangugage.TypeScript;
+        break;
+      case ".mjs":
+        sdkLang = SDKLangugage.TypeScript;
+        break;
+      default:
+        throw new Error("Unsupported file type");
```

Comment:
`throw new Error(`Unsupported file type "${extension}"`);` ?

Reply:
Better one, will do that.